src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -304,9 +304,13 @@ impl Action for IngestAction {
 
         let mut futs = FuturesUnordered::new();
 
-        for _n in 0..self.repeat {
+        for iteration in 0..self.repeat {
             for row in &self.rows {
-                let mut row = row.as_bytes();
+                let row_replaced = row.replace(
+                    &INGEST_ITERATION_RUNTIME_VAR_DECORATED,
+                    &iteration.to_string(),
+                );
```

Comment:
I think you might as well use `substitute_vars` here:

```
use maplit::hashmap;

// ...

let row = substitute_vars(row, hashmap! { "kafka-ingest.iteration".into() => iteration.into() });
```

(The performance doesn't matter and this way we don't hardcode knowledge about the variable syntax in multiple places.)